test/integration/puppetlabs/services/master/environment_classes_int_test.clj

```diff
@@ -690,7 +690,7 @@
              _ (reset! wait-atom {:continue-promise continue-promise
                                   :wait-promise wait-promise})
              initial-response-future (future (get-env-classes "production"))]
-         (is (true? (deref wait-promise 10000 :timed-out))
+         (is (true? (deref wait-promise 20000 :timed-out))
```

Comment:
Does this let it reliably pass? (not that it really matters since it won't stay how it is now)